app/views/submissions/index.html.erb

```diff
@@ -29,6 +29,7 @@
         </td>
         <td><%= submission.urgency %></td>
         <td><%= show_button(submission.person, submission.person&.name.to_s, "fa fa-user-circle") if submission.person %></td>
+        <!-- FIXME: is the logic below correct -->
```

Comment:
I didn't quite understand the logic here?

Reply:
add the submission show button if submission has a person ?
(i might've only needed this with fake data?)

Reply:
Oh, i see. I was confused by the label "Autoemail" on the button:
![Screen Shot 2020-05-23 at 20 10 54](https://user-images.githubusercontent.com/8330/82742917-b4c52a00-9d31-11ea-9583-c8caa10739a8.png)
Seems like the wrong label there?

Reply:
Moved to #407